themes/default/content/blog/pulumi-service-updates/index.md

```diff
@@ -0,0 +1,104 @@
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - anita-trimbur
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - pulumi-service
+    - user-experience
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Earlier this year, Pulumi's user experience (UX) team sat down with users to understand how we can better serve the community. It's important not only to the UX team, but across the company, that we listen to the real wants, needs, pains, and excitements of the people who use our product.
+
+The [Pulumi Service]({{< relref "/product/pulumi-service" >}}) was at the center of many conversations we had with users. The Pulumi Service is the easiest way to use Pulumi’s open source, universal infrastructure as code SDK at scale and provides a fully managed experience. The team is always exploring how we can deliver the best SaaS experience we can. But having input from the community gave us a meaningful point of entry.
+
+<!--more-->
+
+Here are the two problems our users faced that we wanted to improve:
+
+1. Users were losing time trying to get where they needed to go. Important content needs to be quick and easy to find.
+2. Some users didn't feel like they understood the "big picture" of their operations. We wanted to offer the proper sight lines into an organization's IaC.
+
+We knew we could do better. Now, we're excited to share the changes we've made so far!
+
+## New primary navigation pane
+
+Next time you sign in to the Pulumi Service, look for the navigation pane on the left hand side of the page. All primary navigation items like your Dashboard, Stacks, and Settings are accessible in the new pane from anywhere in the Service.
+
+![""](dashboard-nav-expanded.png)
+
+The navigation pane consolidates actions and information so you don't have to look so hard to find where things are. You'll find options in the pane to:
+
+- switch between your individual account and any other organizations you're part of
+- create new organizations
+- access your favorite stacks (read on to find out more!)
+- directly access sub-pages like Billing or Access Tokens
+- send us your feedback!
+
+![""](dashboard-nav-collapsed.png)
+
+We made these changes to help users to move through the Pulumi Service without losing their place and ensure constant access to key pages. Even if you collapse the navigation pane for more screen real estate, you can still get where you need to go!
```

Comment:
can you clarify what is meant by "even if you collapse the pane... you can still get where you need to go"?